Jellyfin.Server/StartupOptions.cs

```diff
@@ -11,6 +11,9 @@ public class StartupOptions : IStartupOptions
         [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (database files, etc.).")]
         public string DataDir { get; set; }
 
+        [Option('C', "configdir", Required = false, HelpText = "Path to use for caching.")]
```

Comment:
--configdir should be --cachedir?

Reply:
Not sure about the use of capital C.  Maybe not quick quick option for cache..

Reply:
Capital C is a fine convention IMO.